src/LudoGameApp/GameEngine/LudoEngine.cs

```diff
@@ -0,0 +1,280 @@
+﻿using System;
+using System.Collections.Generic;
+
+
+namespace GameEngine
+{
+    public class LudoEngine
+    {
+        public int Counter = 0;
+        private int LastDiceThrow { get; set; }
+        private int nrOfPlayer;
+        public bool OkToStart { get; set; }
+        public int NrOfPlayer
+        {
+            get
+            {
+                return nrOfPlayer;
+            }
+            set
+            {
+                if (value < 2 || value > 4)
+                {
+                    OkToStart = false;
+                }
+                else
+                {
+                    OkToStart = true;
+                    nrOfPlayer = value;
+                }
+            }
+        }
+        public List<Player> PlayersList { get; set; }
+
+        public List<Tile> TileList { get; set; }
+        public List<Tile> FinalStretch { get; set; }
+
+        public LudoEngine(int numberOfPlayers)
+        {
+            NrOfPlayer = numberOfPlayers;
+            if (OkToStart)
+            {
```

Comment:
What does this code do?
Should it be in it's own method?
Should it be in the constructor?